packages/react/src/components/createOverlayComponent.tsx

```diff
@@ -47,6 +45,7 @@ export const createOverlayComponent = <
         this.el = document.createElement('div');
       }
       this.handleDismiss = this.handleDismiss.bind(this);
+      this.isDismissing = false;
```

Comment:
We need to move this out of the constructor to get the build to pass:

```typescript
class Overlay extends React.Component<Props> {
    overlay?: OverlayType;
    el!: HTMLDivElement;
    isDismissing: boolean = false;
...
```

Reply:
Gotcha, cool, Ill update!